app/controllers/links_controller.rb

```diff
@@ -29,15 +29,28 @@ def external_url(type:, options: {})
 
   private
 
+  def foreman_org_path(sub_path)
+    "https://theforeman.org/#{sub_path}"
+  end
+
   def documentation_url(section = "", options = {})
-    root_url = options[:root_url] || "https://theforeman.org/manuals/#{SETTINGS[:version].short}/index.html#"
+    root_url = options[:root_url] || foreman_org_path("manuals/#{SETTINGS[:version].short}/index.html#")
     root_url + (section || '')
   end
 
-  def plugin_documentation_url(plugin_name, version: nil, options: {})
-    root_url = options[:root_url] || "https://theforeman.org/plugins"
-    path = version ? "#{plugin_name}/#{version}" : plugin_name
-    "#{root_url}/#{path}"
+  def plugin_documentation_url(options = {})
+    name, version, section, root_url = options.values_at(:name, :version, :section, :root_url)
+
+    path = root_url || foreman_org_path("plugins")
+
+    return path if name.nil?
+
+    path += "/#{name}"
+
+    plugin_version = version.nil? ? Foreman::Plugin.find(name).version.scan(/\d+\.\d+/).first : version
```

Comment:
Added a default to plugin's version, no need so the `version` param isn't always needed.

an idea: we could instead use `latest` and adjust the code in `theforeman/theforeman.org`
in order to match `foreman_discovery/latest` to `foreman_discovery/15.0`

Reply:
This assumes plugins always document with major.minor but this breaks for:
* https://theforeman.org/plugins/foreman_ansible/ (only tracks major versions)
* https://theforeman.org/plugins/foreman_azure/ (could be modified to major.minor)
* https://theforeman.org/plugins/foreman_docker/ (unmaintained, not that interesting)

Also, you can write it as
```suggestion
    plugin_version = version || Foreman::Plugin.find(name).version.scan(/\d+\.\d+/).first
```
* https://theforeman.org/plugins/foreman_kubevirt/ (doesn't appear to have a lot of documentation)

Reply:
> This assumes plugins always document with major.minor but this breaks for..

right, so let's drop it for now.. though maybe as a follow-up with a collaboration of theforeman.org refactoring,
we could just pass the `latest` for version from here, and theforeman.org will know how to route it to the correct version